feature/service-registration/domain/src/main/java/com/pob/umpa/feature/service_registration/domain/sheet_music/entity/SheetMusicServiceProfileEntity.kt

```diff
@@ -0,0 +1,40 @@
+package com.pob.umpa.feature.service_registration.domain.sheet_music.entity
+
+typealias ServiceProfile_SheetMusic = SheetMusicServiceProfileEntity
+
+data class SheetMusicServiceProfileEntity(
+    val serviceProfile: ServiceProfile,
+    val sheet: List<Sheet>,
+)
+
+data class ServiceProfile(
+    val title: String,
+    val image: Image,
+)
+
+data class Image (
+    val url: String,
+    val name: String,
+    val width: Int,
+    val height: Int,
+)
+
+data class Sheet(
+    val sheetType: SheetType,
+    val pricePerPage: PricePerPage,
+)
```

Comment:
pricePerPage는 페이지 당 가격을 의미하는데 PricePerPage 클래스를 보면
`data class PricePerPage(
    val price: Int,
    val page: Int,
)`
페이지당 가격을 의미하는 `price` 와 전체 페이지를 다루는 `page` 속성이 따로 있는데 모두 Sheet 에 관한속성이로 분리를 따로 안시키는게 사용성이 더 좋을거 같아용

```suggestion
data class Sheet(
    val sheetType: SheetType,
    val totalPage: Int
    val pricePerPage: Int,
)
```

Reply:
> pricePerPage는 페이지 당 가격을 의미하는데 PricePerPage 클래스를 보면 `data class PricePerPage( val price: Int, val page: Int, )` 페이지당 가격을 의미하는 `price` 와 전체 페이지를 다루는 `page` 속성이 따로 있는데 모두 Sheet 에 관한속성이로 분리를 따로 안시키는게 사용성이 더 좋을거 같아용

그게 더 좋은 방식인 거 같네욥 감사합니닷